examples/python/arkitscenes/main.py

```diff
@@ -0,0 +1,198 @@
+    for label_info in annotation["data"]:
+        object_id = label_info["objectId"]
+        label = label_info["label"]
+        rotation = np.array(label_info["segments"]["obbAligned"]["normalizedAxes"]).reshape(3, 3)
+        transform = np.array(label_info["segments"]["obbAligned"]["centroid"]).reshape(-1, 3)[0]
+        scale = np.array(label_info["segments"]["obbAligned"]["axesLengths"]).reshape(-1, 3)[0]
+
+        rot = R.from_matrix(rotation)
+        rr.log_obb(
+            f"world/annotations/box-{object_id}-{label}",
+            half_size=scale,
+            position=transform,
+            rotation_q=rot.as_quat(),
+            color=[160, 230, 130, 0],
+            label=label,
+            timeless=True,
+        )
+
+
+def traj_string_to_matrix(traj_string: str) -> Tuple[str, npt.NDArray[np.float64]]:
+    """
+    Converts trajectory string into translation and rotation matrices.
+
+    Args:
+        traj_string: A space-delimited file where each line represents a camera position at a particular timestamp.
+            The file has seven columns:
+            * Column 1: timestamp
+            * Columns 2-4: rotation (axis-angle representation in radians)
+            * Columns 5-7: translation (usually in meters)
+
+    Returns
+    -------
+        tuple: A tuple containing the timestamp (ts) and the transformation matrix (Rt).
+
+    Raises
+    ------
+        AssertionError: If the input string does not contain 7 tokens.
+    """
+    tokens = traj_string.split()  # Split the input string into tokens
+    assert len(tokens) == 7, f"Input string must have 7 tokens, but found {len(tokens)}."
+    ts: str = tokens[0]  # Extract timestamp from the first token
+
+    # Extract rotation from the second to fourth tokens
+    angle_axis = [float(tokens[1]), float(tokens[2]), float(tokens[3])]
+    r_w_to_p = cv2.Rodrigues(np.asarray(angle_axis))[0]
+
+    # Extract translation from the fifth to seventh tokens
+    t_w_to_p = np.asarray([float(tokens[4]), float(tokens[5]), float(tokens[6])])
+
+    # Construct the extrinsics matrix
+    extrinsics = np.eye(4, 4)
+    extrinsics[:3, :3] = r_w_to_p
+    extrinsics[:3, -1] = t_w_to_p
+
+    # Compute the inverse of extrinsics matrix to get the rotation and translation matrices
+    Rt = np.linalg.inv(extrinsics)
+
+    return (ts, Rt)
+
+
+def log_arkit(recording_path: Path) -> None:
+    """
+    Logs ARKit recording data using Rerun.
+
+    Args:
+        recording_path (Path): The path to the ARKit recording.
+
+    Returns
+    -------
+        None
+    """
+    video_id = recording_path.stem
+    image_dir = recording_path / "lowres_wide"
+    depth_dir = recording_path / "lowres_depth"
+    intrinsics_dir = recording_path / "lowres_wide_intrinsics"
+    traj_path = recording_path / "lowres_wide.traj"
+
+    frame_ids = [x.name for x in sorted(depth_dir.iterdir())]
+    frame_ids = [x.split(".png")[0].split("_")[1] for x in frame_ids]
+    frame_ids.sort()
+
```

Comment:
Navigation in the "world" view is a bit tedious without setting an "up direction". For instance: `rr.log_view_coordinates("world", up="+Y", timeless=True)` (I don't actually know what the correct up axis is here)